.drone.yml

```diff
@@ -49,8 +46,6 @@ steps:
   when:
     instance:
     - drone-publish.longhorn.io
-    ref:
```

Comment:
Why removing this part? This only for tags.

Reply:
Is the event `tag` is sufficient to make sure the step only run when there is a change in tag?
```
    event:
     - tag
```

Reply:
OK, let's try it out.